scripts/dream.py

```diff
@@ -255,8 +317,25 @@ def image_writer(image, seed, upscaled=False):
                         results.append([path, formatted_dream_prompt])
                 last_results.append([path, seed])
 
+                if opt.save_progress is not None:
+                    nonlocal progress_video_type
+                    if progress_video_type == 'v' or progress_video_type == 'vo':
+                            frames_for_video.append(image)
+                            make_video(frames_for_video, os.path.join(current_outdir, f'{prefix}.{seed}.mp4'))
+                            frames_for_video.clear()
+
+                if opt.show_progress is not None:
+                    show_progress(image)
+                    cv2.waitKey(1000)
```

Comment:
Remove as you already call waitKey in show_progress?

Reply:
Not the same. The waitKey in show_progress is to keep the window alive. The waitKey here lasts for a second so the user gets a preview of the final output for atleast a second before it moves on -- added this because when a user is performing multiple iterations, the final preview gets taken off screen too quickly.